n2v/inverter.py

```diff
@@ -209,22 +207,26 @@ def invert(self, method,
                      guide_potential_components = ["fermi_amaldi", "svwn"], 
                      opt_max_iter = 50,
                      opt_tol      = 1e-5,
-                     reg=None):
+                     reg=None,
+                     lam=100):
         """
         Handler to all available inversion methods
         """
 
+        self.lam = lam
```

Comment:
Could you add a comment for self.lam?

Reply:
And for other attributes that appears newly?

Reply:
Would you prefer those attributes to appear documented on the `zmp.py` file or under `self.invert()`?

Reply:
I'm not sure. Where is approachable by the help function in Python?

Reply:
Good one. Added docstring on the inverter level. That should be accessible to help.